WalletWasabi.Fluent.Desktop/Extensions/AppBuilderExtension.cs

```diff
@@ -9,7 +9,7 @@ public static class AppBuilderExtension
 {
 	public static AppBuilder SetupAppBuilder(this AppBuilder appBuilder)
 	{
-		bool enableGpu = Services.Config is null ? false : Services.Config.EnableGpu;
+		bool enableGpu = Services.PersistentConfig is null ? false : Services.PersistentConfig.EnableGpu;
```

Comment:
Good question. I think we should not because the Gui uses the settings to display/modify the values of the Config.json file (what we now call PersistentConfig).